lib/github_comment.rb

```diff
@@ -2,31 +2,37 @@
 require 'pr_student'
 
 class GitHubComment
-  AUTH = { :username => ENV["GITHUB"] }
   # def get_comments()
   #   "https://api.github.com/repos/#{ repo_url }/pulls"
   #   response = HTTParty.get(url, headers: {"user-agent" => "rails"}, :basic_auth => GitHubComment::AUTH)
   #   return response
   # end
 
-  def self.find_template(repo)
+  attr_reader :token
+
+  def initialize(token)
+    @token = token
+  end
+
+  def find_template(repo)
     url = "https://raw.githubusercontent.com/#{ repo.repo_url }/master/feedback.md"
-    response = HTTParty.get(url, headers: {"user-agent" => "rails"}, :basic_auth => GitHubComment::AUTH)
+    response = HTTParty.get(url, headers: {"user-agent" => "rails", "Authorization" => "token #{ @token }"})
```

Comment:
Since we have `attr_reader :token` we can just say `token` here to make Sandi Metz happy.